[BLOG]-CNCF(LFX) Industrial EAI-benchmarking with KubeEdge-Ianvs added

RONAK-AI647 opened the pull request:
* **Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)


<!--
*Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
-->

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

**Blog Post
Feature**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
* 
 _Embodied Intelligence meets KubeEdge-Ianvs: Industrial Assembly Benchmarking_

**This blog introduces how to enable comprehensive embodied intelligence benchmarking for industrial manufacturing using the KubeEdge-Ianvs framework.**

RONAK-AI647 committed:
cncf-lfx-EAI-benchmarking with KubeEdge-Ianvs added

Signed-off-by: RONAK <codeitronak226277@gmail.com>

RONAK-AI647 commented:
/hold

RONAK-AI647 committed:
lfx-EAI blog added

Signed-off-by: RONAK <codeitronak226277@gmail.com>

RONAK-AI647 committed:
Update EAI-meets-KubeEdge-Ianvs.mdx

Signed-off-by: RONAK <codeitronak226277@gmail.com>

RONAK-AI647 committed:
add lfx EAI blog

Signed-off-by: RONAK <codeitronak226277@gmail.com>

RONAK-AI647 commented:
@kevin-wangzefeng @Shelley-BaoYue its ready to review , please take a look !!!

fujitatomoya commented on blog/cncf-lfx-EAI-benchmarking/EAI-meets-KubeEdge-Ianvs.mdx:
```diff
@@ -0,0 +1,494 @@
+#### Current Limitations in the Field:
+
+- **Isolated Task Focus**: Popular benchmarks like YCB and OCRTOC evaluate individual capabilities (grasping, object detection) but don't assess how these integrate in multi-stage industrial workflows
+- **Missing Deformable Component Coverage**: While NIST has developed benchmarks for rigid components and simple deformable objects (cables, belts), existing datasets don't support multiple representations of deformable objects with different self-occlusions typical in manufacturing [PubMed Central](https://www.ncbi.nlm.nih.gov/pmc/)
```
https://www.ncbi.nlm.nih.gov/pmc/ redirects me to https://pmc.ncbi.nlm.nih.gov/, so why do we not have https://pmc.ncbi.nlm.nih.gov/ in here instead?

fujitatomoya commented on blog/cncf-lfx-EAI-benchmarking/EAI-meets-KubeEdge-Ianvs.mdx:
```diff
@@ -0,0 +1,494 @@
+- Familiarity with robotic manipulation concepts
+- Understanding of computer vision fundamentals
+
+> **Note**: This benchmark has been tested on Linux platforms. Windows users may need to adapt commands accordingly.
```
if that is not supported on windows, why do we put that required platform more specifically (e.g Ubuntu Noble) for user documentation? so that we can avoid complication and problems in the 1st place.

RONAK-AI647 replied:
https://github.com/kubeedge/ianvs/blob/main/docs/guides/quick-start.md#:~:text=In%20this%20example%2C%20we%20are%20using%20the%20Linux%20platform%20with%20Python%203.8.%20If%20you%20are%20using%20Windows%2C%20most%20steps%20should%20still%20apply%20but%20a%20few%20commands%20and%20package%20requirements%20might%20be%20different.

I included the Windows note because users often try to run benchmarks on Windows, Specifying "Ubuntu Noble" is too restrictive—the benchmark works on Ubuntu 18.04+, Debian, and other Linux distros, if you say I will strict the sentence for the same.

RONAK-AI647 committed:
Update EAI-meets-KubeEdge-Ianvs.mdx

Signed-off-by: RONAK <codeitronak226277@gmail.com>

Shelley-BaoYue reviewed (commented):
/approve 
cc @MooreZheng

MooreZheng reviewed (changes requested):
Overall it look cool! It is very close to the final version. As discussed on the routine meeting, we can change the storyline a little bit by moving the long tutorial to the end the blog or use a extra link to the project readme. The pictures and videos can also be put forward to make the blog more attracted.

RONAK-AI647 committed:
Update EAI-meets-KubeEdge-Ianvs.mdx

Signed-off-by: RONAK <codeitronak226277@gmail.com>

RONAK-AI647 committed:
Merge branch 'master' into RONAK-A1647/Blog(Embodied-Intelligence-meets-KubeEdge-Ianvs-Industrial-Assembly-Benchmarking)

RONAK-AI647 commented:
cc @MooreZheng  @hsj576 
let me know if any other changes.

https://github.com/user-attachments/assets/da7e984c-2a26-4473-b1c5-d7b21326ae6c

1. Added Pics and link at the very start 
2. Moved the tutorial to the end with Readme link.

MooreZheng reviewed (commented):
/lgtm

hsj576 commented:
/lgtm
